### PR TITLE
added the pypiwin32 package

### PR DESCRIPTION
**Pull request by xiaomaogy**

### CODE REVIEW

File: ml-agents/tests/trainers/test_ppo.py [hunk 1]
```diff
@@ -42,15 +42,15 @@ def dummy_config():
 
 @mock.patch('mlagents.envs.UnityEnvironment.executable_launcher')
 @mock.patch('mlagents.envs.UnityEnvironment.get_communicator')
-def test_ppo_policy_evaluate(mock_communicator, mock_launcher):
+def test_ppo_policy_evaluate(mock_communicator, mock_launcher, dummy_config):
```
Comment (vincentpierre) [inline ml-agents/tests/trainers/test_ppo.py]: Has this been tested on both Windows and Mac ?

File: docs/Installation-Windows.md [hunk 2]
```diff
@@ -153,6 +153,12 @@ pip install -e .
 This will complete the installation of all the required Python packages to run
 the ML-Agents toolkit.
 
+If you are stuck during this installation, you can try:
```
Comment (vincentpierre) [inline docs/Installation-Windows.md]: Maybe adding a little more description of what being "stuck" is like and add a few lines describing what `--no-cache-dir` does.

File: ml-agents/setup.py [hunk 3]
```diff
@@ -37,7 +37,8 @@
         'docopt',
         'pyyaml',
         'protobuf>=3.6,<3.7',
-        'grpcio>=1.11.0,<1.12.0'],
+        'grpcio>=1.11.0,<1.12.0',
+        'pypiwin32==223;platform_system=="Windows"'],
```
Comment (vincentpierre) [inline ml-agents/setup.py]: Was this tested on both Mac and Windows ?

**Comment by vincentpierre** [comment]
I think the documentation change could use a little more description. If the changes to setup and the tests have been tested, I see no problem with them and I think we can merge them.

File: ml-agents/setup.py (hunk 3 above)
Comment (xiaomaogy) [reply ml-agents/setup.py]: Yes I tested this on Mac, Windows and Linux.

File: docs/Installation-Windows.md (hunk 2 above)
Comment (xiaomaogy) [reply docs/Installation-Windows.md]: Will change.

File: ml-agents/tests/trainers/test_ppo.py (hunk 1 above)
Comment (xiaomaogy) [reply ml-agents/tests/trainers/test_ppo.py]: Yes.

**Pull request merged**